components/contact/ContactSectionRow.js

```diff
@@ -1,18 +1,23 @@
 import { Button } from '@dts-stn/service-canada-design-system'
 import { Fragment } from 'react'
 import Markdown from 'markdown-to-jsx'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faClock } from '@fortawesome/free-solid-svg-icons'
 
 function ContactSectionRow(props) {
-  const { label, detail, index, highlight, button } = props
+  const { label, detail, index, highlight, icon, button } = props
   return label && detail ? (
     <div
-      className={`grid grid-cols-5 gap-4 border-t-2 mt-4 ${
+      className={`grid grid-cols-12 gap-4 border-t-2 my-4 p-2 ${
         highlight && 'bg-blue-100'
       }`}
       key={index}
     >
-      <div className="col-span-1 font-bold text-xl">{label}</div>
-      <div className="col-span-4 [&_ul]:list-inside [&_ul]:ml-4 [&_ul]:list-disc">
+      <div className="col-span-3 font-bold text-xl">{label}</div>
+      <div className="col-span-1 grid justify-center m-2">
+        {icon && <FontAwesomeIcon icon={faClock} />}
```

Comment:
Icon looks good but is there a way to make this more dynamic? We're assuming the only icon we'll ever need is faClock which is fine just might want to clarify that this is the case with content since the field in AEM is currently for any icon.

Also we may need to pull in an icon bg-colour from AEM to avoid doing that logic on the page.

Reply:
I debated this. The problem is that the icon library can be inconsistent, and it would be fairly easy for someone to submit an icon definition that's valid under FA, but still fails in the app. We would need to constrain the choices of icons that we'll accept from AEM.

Reply:
Take a look at how the icons are done for the BenefitTasks and add the icon to the loadicons.js list. this way any icon can be used.

Reply:
Done.